package/MDAnalysis/transformations/fit.py

```diff
@@ -0,0 +1,206 @@
+    
+    Example
+    -------
+    Removing the translations of a given AtomGroup `ag` on the XY plane by fitting 
+    its center of mass to the center of mass of a reference `ref`:
+    
+    .. code-block:: python
+    
+        ag = u.select_atoms("protein")
+        ref = mda.Universe("reference.pdb")
+        transform = mda.transformations.fit_translation(ag, ref, plane="xy",
+                                                        center_of="mass")
+        u.trajectory.add_transformations(transform)
+    
+    Parameters
+    ----------
+    ag : Universe or AtomGroup
+       structure to translate, a
+       :class:`~MDAnalysis.core.groups.AtomGroup` or a whole 
+       :class:`~MDAnalysis.core.universe.Universe`
+    reference : Universe or AtomGroup
+       reference structure, a :class:`~MDAnalysis.core.groups.AtomGroup` or a whole 
+       :class:`~MDAnalysis.core.universe.Universe`
+    plane: str, optional
+        used to define the plane on which the translations will be removed. Defined as a 
+        string of the plane. Suported planes are yz, xz and xy planes.
+    weights : {"mass", ``None``} or array_like, optional
+       choose weights. With ``"mass"`` uses masses as weights; with ``None``
+       weigh each atom equally. If a float array of the same length as
+       `ag` is provided, use each element of the `array_like` as a
+       weight for the corresponding atom in `ag`.
+    
+    Returns
+    -------
+    MDAnalysis.coordinates.base.Timestep
+    """
+    
+    if plane is not None:
+        axes = {'yz' : 0, 'xz' : 1, 'xy' : 2}
+        try:
+            plane = axes[plane]
+        except (TypeError, KeyError):
+            raise ValueError('{} is not a valid plane'.format(plane))
+    try:
+        if ag.atoms.n_residues != reference.atoms.n_residues:
+            raise ValueError("{} and {} have mismatched number of residues".format(ag,reference))
+    except AttributeError:
+        raise AttributeError("{} or {} is not valid Universe/AtomGroup".format(ag,reference))
+    ref, mobile = align.get_matching_atoms(reference.atoms, ag.atoms)
+    try:
+        weights = align.get_weights(ref.atoms, weights=weights)
+    except (ValueError, TypeError):
+        raise ValueError("weights must be {'mass', None} or an iterable of the "
+                        "same size as the atomgroup.")
+    
+    ref_com = np.asarray(ref.center(weights), np.float32)
+
+    def wrapped(ts):
+        mobile_com = np.asarray(mobile.atoms.center(weights), np.float32)
+        vector = ref_com - mobile_com
+        if plane is not None:
+            vector[plane] = 0
+        ts.positions += vector
+        
+        return ts
+    
+    return wrapped
+
+
+def fit_rot_trans(ag, reference, plane=None, weights=None):
+    """Perform a spatial superposition by minimizing the RMSD.
+
+    Spatially align the group of atoms `ag` to `reference` by doing a RMSD
+    fit on `select` atoms.
```

Comment:
I'm not sure what `select` refers to here--maybe an old argument for sub-selections that was removed?